server_test.go

```diff
@@ -674,10 +675,12 @@ func TestGlobalServerS3PluginFlush(t *testing.T) {
 
 	s3p := &s3p.S3Plugin{Logger: log, Svc: client}
 
+	pluginCount := len(server.getPlugins())
```

Comment:
Is there a reason we're checking this number dynamically (based on the length) rather than statically (ie, the number of plugins we expect to have in the end)? Our test runs should be independent, so we should know ahead-of-time the number of plugins that will be loaded.